github/changedDirectories.go

```diff
@@ -62,7 +62,7 @@ func GetChangedProjects(repo *git.Repository, headRef, baseRef string) (files []
 		}
 		dir := getTopLevelDirName(name)
 		if dir != "" {
-			files = appendIfMissing(files, dir)
+			directories = appendIfMissing(directories, dir)
```

Comment:
Still not a fan of this. Working with unique strings is just asking to be a set, not a list (slice). Not a deal breaker though.